src/main/java/com/cdancy/bitbucket/rest/features/FileApi.java

```diff
@@ -94,14 +94,16 @@ Commit updateContent(@PathParam("project") String project,
                          @Nullable @FormParam("sourceBranch") String sourceBranch);
 
     @Named("file:list-files")
-    @Documentation({"https://developer.atlassian.com/static/rest/bitbucket-server/5.1.0/bitbucket-rest.html#idm45588159557712"})
+    @RequestFilters(ScrubNullFromPathFilter.class)
```

Comment:
I see this is coming in handy :)

Reply:
Yes! There are several APIs following that format